test_runner/regress/test_config.py

```diff
@@ -49,9 +49,9 @@ def test_safekeepers_reconfigure_reorder(
     old_sks = ""
     with closing(endpoint.connect()) as conn:
         with conn.cursor() as cur:
-            cur.execute("SHOW neon.safekeepers")
+            cur.execute("SHOW neon.safekeeper_connstrings")
```

Comment:
I suppose we would need some tests where both the old and the new settings are defined, and check which of these has been taken into account then?